run-tests.php

```diff
@@ -2078,21 +2078,6 @@ function run_test($php, $file, $env)
 			}
 			$wanted_re = $temp;
 
-			$wanted_re = str_replace(
-				array('%binary_string_optional%'),
-				'string',
-				$wanted_re
-			);
-			$wanted_re = str_replace(
-				array('%unicode_string_optional%'),
-				'string',
-				$wanted_re
-			);
-			$wanted_re = str_replace(
-				array('%unicode\|string%', '%string\|unicode%'),
-				'string',
-				$wanted_re
-			);
 			$wanted_re = str_replace(
 				array('%u\|b%', '%b\|u%'),
```

Comment:
We can drop this one as well.

Reply:
There're considerable cases, gonna work on that so :construction: